app/parse.py

```diff
@@ -8,8 +15,55 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def parse_one_quote(quote_soup: BeautifulSoup) -> Quote:
+    text = quote_soup.select_one(".text").text
+    author = quote_soup.select_one(".author").text
+    tags = quote_soup.select_one(".keywords")
+
+    if tags and tags.has_attr("content"):
```

Comment:
The parse_one_quote function handles tags in a slightly convoluted way. It first checks if the tags element exists and has a content attribute. This can be simplified to reduce redundancy and make the code more readable.